src/main/webapp/app/exam/manage/exercise-groups/exercise-groups.component.html

```diff
@@ -132,12 +132,20 @@ <h5 class="group-title font-weight-bold mb-0">{{ exerciseGroup.title }}</h5>
                                         <button
                                             jhiDeleteButton
                                             [entityTitle]="exerciseGroup.title || ''"
-                                            deleteQuestion="artemisApp.examManagement.exerciseGroup.delete.question"
+                                            deleteQuestion="{{
+                                                localVCEnabled
```

Comment:
Should we also check here that programming exercises are actually present in the exercise group? Otherwise it could be a bit confusing when trying to delete a group with modelling exercises for example